src/menu/menu-item.jsx

```diff
@@ -83,6 +83,14 @@ const MenuItem = React.createClass({
   },
 
   getStyles() {
+    const isRtl = this.context.muiTheme.isRtl;
```

Comment:
Fixed. This was written before `state.muiTheme` was introduced. I've just seen that feature.